### ISSUE TITLE
Breakpoints Sometimes not resolving with Node 6.1.0

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/nodejstools#934]
**Repo**

1. Install Node 6.1.0
2. Create new app.
3. Create new file (seems to repo more in larger projects and files besides the main file).
3. Set some breakpoints this new file.
4. `require` the file from main.
5. Try debugging.

**Expected**
Debugger stops at breakpoints

**Actual**
Breakpoints are not resolved.

Breakpoints were already less stable with Node 6.0.0, but with 6.1.0 I'm not seeing them resolved at all. Node 5.x branches work fine.

https://github.com/Microsoft/vscode/issues/6029

**Comment by NoelAbrahams** [comment]
Any idea when this is likely to be resolved? This issue is blocking our adoption of Node 6.1 so would appreciate a fix soon. Thanks.

**Comment by mjbvz** [comment]
I'm looking into this, but right now it is not clear what changed on the Node side of things or what the root cause of the problem is.

**Comment by mjbvz** [comment]
Quick update: The root cause seems to be a change on the node side of things, not in NTVS directly , but it is possible we were relying on bugged behavior previously. The issue is nondeterministic and more common on larger projects. I believe it also exists in Node 6.0.

I'm seeing several different behaviors:
* Sometimes everything works.
* Sometimes breakpoints are not resolved.
* Sometimes breakpoints are resolved but then cause node to emit `Internal error: illegal access"` when hit and we call `backtrace`.

https://github.com/node-inspector/node-inspector/issues/864

**Comment by mjbvz** [comment]
Update on some of the earlier leads:

The path differences don't seem to matter. I tried using a case insensitive regex to match the file name when setting the initial breakpoint, and using a file path with no case differences except for the drive letter. These seem unrelated to the breakpoint being hit or not.

Also, I see the backtrack trigger `Internal error: illegal access` both when breakpoints are hit and when they are not. Setting the breakpoint inside of the express router seems to be a reliable trigger:

```js
router.get('/', function (req, res) {
    res.render('index', { title: 'Express' }); // set breakpoint here
});
```

**Comment by mousetraps** [comment]
This fix is available in v1.2 beta. Please try it out and let us know what you think! https://aka.ms/ntvslatest

**Comment by NoelAbrahams** [comment]
We've been trying out 1.2 beta with Node 6.2 and the breakpoints seem to be working so far. Thanks.
